internal-api/src/main/java/datadog/trace/api/iast/telemetry/IastMetric.java

```diff
@@ -18,7 +18,8 @@ public enum IastMetric {
       "executed.sink", true, Scope.REQUEST, Tag.VULNERABILITY_TYPE, Verbosity.INFORMATION),
   EXECUTED_TAINTED("executed.tainted", true, Scope.REQUEST, Verbosity.DEBUG),
   REQUEST_TAINTED("request.tainted", true, Scope.REQUEST, Verbosity.INFORMATION),
-  TAINTED_FLAT_MODE("tainted.flat.mode", false, Scope.GLOBAL, Verbosity.INFORMATION);
+  TAINTED_FLAT_MODE("tainted.flat.mode", false, Scope.GLOBAL, Verbosity.INFORMATION),
+  JSON_TAG_SIZE_EXCEED("json.tag.size.exceed", false, Scope.GLOBAL, Verbosity.INFORMATION);
```

Comment:
```suggestion
  JSON_TAG_SIZE_EXCEED("json.tag.size.exceeded", false, Scope.GLOBAL, Verbosity.INFORMATION);
```

I would probably used _exceeded_ for clarity, but LGTM.